src/domain/app/header/userMenu/UserMenu.tsx

```diff
@@ -18,7 +18,7 @@ const UserMenu: React.FunctionComponent<UserMenuProps> = ({ options }) => {
       {options.map((option, index) => {
         return (
           <li key={index} onClick={() => option.onClick && option.onClick()}>
-            {option.label}
+            <button>{option.label}</button>
```

Comment:
I'm pretty sure it would be more "correct" to have a link here instead of a button, but that improvement requires a refactoring of the whole user menu. I propose we spend time on that if someone complains, the site should be usable as it is!